src/sdl_texture_mjpg.cpp

```diff
@@ -14,12 +15,20 @@ SDLTextureMJPG::~SDLTextureMJPG() {
   free(rgbdata_);
 }
 
-void SDLTextureMJPG::decompress(const unsigned char* jpeg,
-                                unsigned long jpeg_size) {
+int SDLTextureMJPG::decompress(const unsigned char* jpeg,
```

Comment:
This is now "const Span<uint8_t> &data" instead, see latest version of libcamera patch

Reply:
Thanks Eric, I've now based changes off the latest version of the Libcamera patch